tests/elements-regression/src/utils.js

```diff
@@ -1,3 +1,33 @@
 module.exports = {
 	isObject: ( value ) => value && 'object' === typeof value && value.constructor === Object,
+
+	/**
+	 * @param {Object} tested   e.g. { heading: { title: 'Hello World' } }
+	 * @param {Object} expected e.g. { heading: { controls: { title: 'Hello World' } } }
+	 * @return {{
+	 * 	testedWidgetsCount: number,
+	 * 	expectedWidgetsCount: number,
+	 * 	testedControlsCount: number,
+	 * 	expectedControlsCount: number,
+	 * }}
+	 */
+	summary: ( tested, expected ) => {
```

Comment:
Maybe we should use a custom reporter alongside the regular reporter?
https://playwright.dev/docs/test-reporters#custom-reporters